ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/mapper/InterestMapper.java

```diff
@@ -29,7 +32,7 @@ public void to(Interest interest, CCDCase.CCDCaseBuilder builder) {
             return;
         }
 
-        interest.getSpecificDailyAmount().ifPresent(builder::interestSpecificDailyAmount);
+        builder.interestSpecificDailyAmount(interest.getSpecificDailyAmount().map(moneyMapper::to).orElse(null));
```

Comment:
Suggestion - interest.getSpecificDailyAmount().map(moneyMapper::to)
            .ifPresent(builder::interestSpecificDailyAmount);

Reply:
The above looks more readable.

Reply:
done